pyproject.toml

```diff
@@ -0,0 +1,129 @@
+[project]
+name = "robotframework-robocop"
+description = "Static code analysis tool (linter) and code formatter for Robot Framework"
+readme = "README.md"
+authors = [
+  { name = "Bartlomiej Hirsz", email = "bartek.hirsz@gmail.com" },
+  { name = "Mateusz Nojek", email = "matnojek@gmail.com" },
+]
+requires-python = "~=3.9,<3.13"
+dependencies = [
+    "jinja2>=3.1.4,<4.0",
+    "robotframework>=4.0,<8.0",
```

Comment:
Thinking if supporting RF 4.0 makes sense. Of course, there's always someone who uses it, but... Wouldn't it make it harder to maintain the tool?
Such decision at the beginning can give more freedom and simplify the maintenance.

Reply:
We are already 'jumping' from 3.2.2 (for robocop linter) and it will bring benefits in ease of maintenance. Since it will be 'breaking' version, I don't see reason why not jump to higher version altogether. What would be best version to use as min?

Reply:
Sure, 4.0.0 is still higher than 3.2.2, so I think it makes sense to keep it as min.